sql/codegen.go

```diff
@@ -375,15 +378,15 @@ classifier.compile(optimizer=classifier.default_optimizer(),
     metrics=["accuracy"])
 classifier.fit(input_fn(BATCHSIZE, is_train=True),
     epochs=EPOCHS if EPOCHS else classifier.default_training_epochs(),
-    verbose=0)
+    verbose=VERBOSE)
```

Comment:
This only works for keras models, can set `tf.get_logger().setLevel(logging.INFO)` if `verbose>1`?

Reply:
Good point, will do that in the next pr.